### ISSUE TITLE
Validation of translated model with Ardent

### ISSUE THREAD

**Issue by mdiniz** [dimsav/laravel-translatable#31]
Hi there

I just got Translatable working with Ardent with the help of your gist but I'm facing an issue. 

To use your example with Country, both Country and CountryTranslation would extend Ardent. I would like to validate the name of the country (say, max 100 characters). For that I would configure the validator in CountryTranslation. I'm able to save translations when the name is valid. So far so good.

If I do something like this:

```
$country->translate('en')->name = 'a very long name with more than 100 chars';
$country->save();
```

Then CountryTranslation doesn't save (which is right, since it's not valid). However, I'm unable to get the error messages. If I do `$country->errors()->all()`, the list of errors is empty. Also, `$country->validate()` returns true, which is weird.

Do you have any idea about this? That's probably not an issue with Translatable itself, just the way it integrates with Ardent.

**Comment by mvdstam** [comment]
I've come up with a solution, by simply coming up with a base model that both extends Ardent and uses the Translatable trait: https://github.com/mvdstam/ardent-translatable/.

It's also on Packagist, but use it on your own risk :smiley:. I've done some preliminary testing and everything seems to play pretty nice with each other. Using this base class also fixes #12.

**Comment by dimsav** [comment]
:+1: @mvdstam I will put this on the readme file as soon as it is stable.
